README.md

```diff
@@ -1 +1,148 @@
+  </ul>
+</details>
+
+<details>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Social_Media_links]**
+- **[navigation_bar]**
+- **[home_page]**
+- **[about_page]**
+- **[main_program_section]**
+- **[featured_artists_section]**
+- **[partners_section]**
+- **[footer]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://amen-musingarimi.github.io/musical-concert-capstone/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Setup
+
+Clone this repository to your desired folder using the following command; git clone https://github.com/Amen-Musingarimi/musical-concert-capstone.git
+
+### Run tests
+
+To run tests, run the following command: npx hint . and npx stylelint "\*_/_.{css,scss}" and npx eslint .
+
+### Deployment
+
+You can deploy this project using Github pages.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Amen Musingarimi**
+
+- GitHub: [@Amen-Musingarimi](https://github.com/Amen-Musingarimi)
+- Twitter: [@Amen-Musingarimi](https://twitter.com/Amen-Musingarimi)
+- LinkedIn: [atmusingarimi](https://www.linkedin.com/in/atmusingarimi/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[nav_bar]**
+- [ ] **[home_page]**
+- [ ] **[about_page]**
+- [ ] **[featured_artist_page]**
+- [ ] **[footer]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Amen-Musingarimi/musical-concert-capstone/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project then don't forget to give a star ⭐ on this repository.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank [Cindy Shin](https://www.behance.net/adagio07), the author of the original [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](LICENSE.md) licensed.
```

Comment:
- [Optional] good job by adding a link to the home page as the last reviewer said but I think it would be better to make the link match the design you can just add The word Home instead.
![image](https://user-images.githubusercontent.com/74744787/217658271-c943f23d-833d-4e62-be12-11f83580b227.png)